### PR TITLE
fix(w3c/headers): mismatched title warning

### PR DESCRIPTION
**Pull request by Swapnilr1**
This is a continuation to #2251 fixing a small bug with the implementation, when both `h1` and `document.title` are present and `isPreview`, `prUrl`, and `prNumber` are set, then ReSpec emits a warning `The document's title and the "<title>" element differ.` due to whitespace being present.

Sorry for not testing this initially. 

I don't think I can have a test for this until #1084 is fixed.

### CODE REVIEW

File: src/w3c/templates/headers.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ function getSpecTitleElem(conf) {
     `;
     specTitleElem.prepend(...childNodes);
   }
-  conf.title = specTitleElem.textContent.trim();
+  conf.title = specTitleElem.textContent.replace(/\s{2,}/g, " ").trim();
```
Comment (marcoscaceres) [inline src/w3c/templates/headers.js]: Probably want `/\s\s+/g` here, no? Just normalizing whitespace?  

```suggestion
  conf.title = specTitleElem.textContent.replace(/\s{2,}/g, " ").trim();
```

Comment (sidvishnoi) [reply src/w3c/templates/headers.js]: We have a `norm` in utils.js

Comment (marcoscaceres) [reply src/w3c/templates/headers.js]: even better!

Comment (Swapnilr1) [reply src/w3c/templates/headers.js]: @marcoscaceres Sorry, what's the difference between `/\s{2,}` and `/\s\s+`? 
@sidvishnoi Yes, we can use that. Thanks.

**Comment by Swapnilr1** [comment]
I hope there aren't any other issues with #2251.

**Comment by marcoscaceres** [comment]
"Only where there is no code there is no bugs" - ancient proverb

File: src/w3c/templates/headers.js (hunk 1 above)
Comment (marcoscaceres) [reply src/w3c/templates/headers.js]: > Sorry, what's the difference between /\s{2,} and /\s\s+?
I guess they are the same... just more used to seeing the other.

**Pull request merged**